presto-mysql/src/test/java/io/prestosql/plugin/mysql/TestMySqlTypeMapping.java

```diff
@@ -371,6 +377,45 @@ public void testDecimalExceedingPrecisionMaxWithSupportedValues(int typePrecisio
         }
     }
 
+    @Test
+    public void testVarbinary()
+    {
+        varbinaryTestCases(mysqlBinaryDataType("varbinary(50)"))
+                .execute(getQueryRunner(), mysqlCreateAndInsert("tpch.test_varbinary"));
+
+        binaryTestCases(mysqlBinaryDataType("binary(50)"))
+                .execute(getQueryRunner(), mysqlCreateAndInsert("tpch.test_varbinary"));
+
+        // maps to LONGVARBINARY
+        varbinaryTestCases(mysqlBinaryDataType("mediumblob"))
```

Comment:
Shall we cover all of these: TINYBLOB, BLOB, MEDIUMBLOB, and LONGBLOB?

Reply:
Sure. I've added them. Figured that since they both mapped to either VARBINARY OR LONGVARBINARY, testing them explicitly would be redudant.